reana_server/rest/secrets.py

```diff
@@ -112,7 +112,10 @@ def add_secrets():  # noqa
               }
     """
     try:
-        user = get_user_from_token(request.args.get("access_token"))
+        if current_user.is_authenticated:
```

Comment:
Can we use the `login_required` decorator? Otherwise, we need to return a 403 if the user is not authenticated nor with the session nor with the token (before we were throwing a [`ValueError` on `get_user_from_token`](https://github.com/reanahub/reana-server/blob/d4cfc0a8fc3930fda3af04647c4bcee0b5a706ba/reana_server/utils.py#L47), catching it and [throwing the 403](https://github.com/reanahub/reana-server/blob/d4cfc0a8fc3930fda3af04647c4bcee0b5a706ba/reana_server/rest/workflows.py#L166)).